<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1817</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/icon.ico</normaloff>:/new/prefix1/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>— 站内监控画面 —</string>
    </property>
   </widget>
   <widget class="QLabel" name="waitqueue_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>610</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>———— 站内排队队列 ————</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="waitqueue_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>651</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>车牌号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>已等待时长(秒)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="parkinglotGraphics">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pkTwoGraphics">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="income">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>100</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Berlin Sans FB&quot;;</string>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="income_label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>收入额</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>650</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>刷卡进站</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>40</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="remain_label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>160</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>余车位数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leaveBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>780</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>结算出站</string>
    </property>
   </widget>
   <widget class="QLabel" name="nowlist_label">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>230</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 25pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>—— 在站车辆列表 ——</string>
    </property>
   </widget>
   <widget class="QLabel" name="remain">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>160</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Berlin Sans FB&quot;;</string>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="nowlist_table">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>310</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>651</width>
      <height>0</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>车牌号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>入站时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>已停时长(秒)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>费用(元)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="announce_label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>—— 公告牌 ——</string>
    </property>
   </widget>
   <widget class="QLabel" name="carGraphics">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1817</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommand">
    <property name="title">
     <string>操作菜单</string>
    </property>
    <addaction name="actionUser"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCommand"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionUser">
   <property name="text">
    <string>用户查询工具</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
